src/Entity/Event/CauseEvent.php

```diff
@@ -30,47 +26,24 @@ class CauseEvent extends BaseEvent
     use DefaultCategoryOwnerTrait;
 
     /**
-     * @ORM\ManyToMany(targetEntity="App\Entity\Coalition\Cause", inversedBy="events")
-     * @ORM\JoinTable(name="event_cause",
-     *     joinColumns={@ORM\JoinColumn(name="event_id", referencedColumnName="id", onDelete="CASCADE", unique=true)},
-     *     inverseJoinColumns={@ORM\JoinColumn(name="cause_id", referencedColumnName="id", onDelete="CASCADE")}
-     * )
+     * @ORM\ManyToOne(targetEntity="App\Entity\Coalition\Cause", inversedBy="events")
      *
-     * @Assert\Count(min=1, max=1, exactMessage="cause_event.cause.invalid")
```

Comment:
We should keep the validation this field, we can add `@NotBlank` constraint, wdyt?